docs/blog/2022-community-update.rst

```diff
@@ -0,0 +1,125 @@
+.. post:: Aug 22, 2022
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - August 2022
+=============================================
+
+Hey there documentarians!
+
+Eric Holscher here, one of the co-founders of Write the Docs.
+Filling in again for the newsletter team in August this year,
+we have a few updates from the community and what has been going on.
+Thanks again to `Aaron <https://www.aaronrcollier.com/>`_ for taking over the newsletter editor role,
+and to `Beth <https://bethaitman.com/>`_ for doing it for the last few years.
+
+We're looking forward to seeing everyone online for our virtual `Prague <http://www.writethedocs.org/conf/prague/2022/>`_ on **September 11-13** in the CEST timezone.
+We're also getting excited about our upcoming virtual `Australia & India <http://www.writethedocs.org/conf/australia/2022/>`_ conference on **December 8-9** in the AEDT timezone.
+The talks were just announced for the Prague conference, and the `Australia conference CFP <https://www.writethedocs.org/conf/australia/2022/cfp/>`_ **is open until September 16 if you have ideas to present.**
```

Comment:
```suggestion
The `Australia conference CFP <https://www.writethedocs.org/conf/australia/2022/cfp/>`_ **is open until September 16 if you have ideas to present.**
```

Reply:
Don't think we need to repeat that the talks for Prague are available since it's already 2 blog posts ago, and might take away from the signposting of the open CFP for AUS.